backend/tests/Designer.Tests/Controllers/DeploymentsControllerTests.cs

```diff
@@ -99,20 +96,21 @@ public async Task GetDeployments_SingleLaggingDeployments_PipelineServiceCalled(
             Assert.Equal(HttpStatusCode.OK, res.StatusCode);
             Assert.Equal(8, actual.Count());
             Assert.Contains(actual, r => r.Build.Status == BuildStatus.InProgress);
-            _pipelineServiceMock.Verify(p => p.UpdateDeploymentStatus(It.IsAny<string>(), It.IsAny<string>()), Times.Once);
+            _deploymentServiceMock.Verify(p => p.UpdateAsync(It.IsAny<string>(), It.IsAny<string>()), Times.Once);
             _deploymentServiceMock.Verify(r => r.GetAsync(_org, _app, It.IsAny<DocumentQueryModel>()), Times.Once);
         }
 
         private List<DeploymentEntity> GetDeploymentsList(string filename)
         {
             string path = Path.Combine(UnitTestsFolder, "..", "..", "..", "_TestData", "Deployments", filename);
-            if (File.Exists(path))
+            if (!File.Exists(path))
             {
-                string deployments = File.ReadAllText(path);
-                return JsonSerializer.Deserialize<List<DeploymentEntity>>(deployments, _options);
+                return null;
             }
 
-            return null;
+            string deployments = File.ReadAllText(path);
+            return JsonSerializer.Deserialize<List<DeploymentEntity>>(deployments, _options);
```

Comment:
This is just nitpick with a healthy does of personal preferences so this is up for discussion.
Is there a reason for changing this? I find ifs without the ! better as it is easy to miss the negation.

Reply:
Suggestion from Rider - I have no preferences here whatsoever. Rider says the following: `Invert 'if' statement to reduce nesting` 🤷🏼‍♀️

Reply:
Trust Rider not me, smarter people have put their thought into that ;)